README.rdoc

```diff
@@ -66,6 +66,12 @@ The plugin supports the following parameters:
                             forwarded. If not set, incomplete exceptions stacks
                             are not flushed.
 
+[add_line_break]  Force the addition of a line break '\n' between
+                  each lines. This is useful if your exception is formatted
+                  as a single line. i.e., logs retrieved from the docker's
```

Comment:
For clarification, how about  `Force line breaks between each lines when combining exception stacks.` instead of `Force the addition of a line break '\n' between each lines.`?